app/routes/events.js

```diff
@@ -1,7 +1,33 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+	toggleEvents : function(buttonClass, eventClass) {
+		if(jQuery(buttonClass).hasClass('disabled')) {
+			jQuery(eventClass).parent().parent().show();
+			jQuery(buttonClass).removeClass('disabled');
+		} else {
+			jQuery(eventClass).parent().parent().hide();
+			jQuery(buttonClass).addClass('disabled');
+		}
+	},
 	model: function() {
 		return this.store.findAll('event');
+	},
+	actions : {
+		filterAll : function(){
+			this.toggleEvents('.pill-triconf', '.event');
```

Comment:
This should be moved to the controller and should instead set properties -- such as `isMusicEventsHidden` -- that are toggled -- `this.set('isMusicEventsHidden', !this.get('isMusicEventsHidden'))` -- then in the template do something like `<div disabled={{if isMusicEventsHidden true false}}>...</div>`. Hope this makes sense, accessing jQuery is almost always the wrong solution unless you are wrapping a jQuery library.